keyext.slicing/src/main/java/org/key_project/slicing/ui/SlicingLeftPanel.java

```diff
@@ -287,6 +291,7 @@ private JPanel getDependencyGraphPanel() {
         panel1.setBorder(new TitledBorder("Dependency graph"));
 
         abbreviateFormulas = new JCheckBox("Abbreviate formulas");
+        abbreviateChains = new JCheckBox("Shorten long chains");
```

Comment:
I think this option is not self-explanatory. In particular, I wonder what the label of the new edge is if multiple edges are contracted to a single one. Could you provide here (and maybe also for "abbreviate formulas")?

Reply:
I'll add some tooltips. Maybe a picture in key-docs too. For now, the edge label is simple the label of the last link in the chain.

For example `A  --1-->  B  --2-->  C  --3-->  D` would shorten to `A  --1-->  B  --3--> D`

Reply:
We discussed that `A  --1-->  B  --2-->  C  --3-->  D` should shorten to `A  --1-->  B  --2, ..., 3-> D`. I'll implement this change now.

Reply:
This is now implemented :)